src/docs/patterns/navigation.mdx

```diff
@@ -22,20 +37,23 @@ Navigation is the way in which users access the different areas of your applicat
 
 ## Fixed Navigation
```

Comment:
Should the top-level variations be "Drawer Navigtaion" and "Tab Navigation"? And then the four variations underneath drawer are Permanent, Persistent, Temporary, and Rail?

Reply:
Split them out into separate sections